Neos.Fusion/Classes/Core/ObjectTreeParser/Exception/ParserException.php

```diff
@@ -18,6 +18,7 @@
 
 /**
  * 'Fluent' exception for the Fusion Parser.
+ * @internal although ${exception.headingMessagePart} ${exception.asciiPreviewMessagePart} and ${exception.helperMessagePart} are used outside for custom error display
```

Comment:
Then it ain't internal, I'd argue. But it's perfectly fine (even necessary) for parser exceptions to be API imho.

Reply:
If the runtime and parser are internal you shouldn't have any reason to try/catch this in userland code I guess?